src/callbacks/onInteraction.md

```diff
@@ -1,27 +1,29 @@
 # $onInteraction
 Triggered upon an interaction being emitted (e.g. a button being clicked).
 
-## Usages
-- `$onInteraction` - Command runs upon any interaction being emitted.
-> **Note:** `$onInteraction` (without brackets) shall only be used once. As it replies directly to the Discord's interaction event--once it processes the data, such data becomes invalid.
-- `$onInteraction[customID]` - Command runs upon the provided 'customID' being clicked/selected/interacted with.
+## Syntax
+```
+$onInteraction
+```
+### Supports
+- **[Buttons](../guides/buttons.md)**
+- **[Select Menu](../guides/selectmenu.md)**
+- **[Modals](../guides/modals.md)**
+
```

Comment:
Add a mention about `$customID`. It is important to properly tell users that they need to use `$customID` to differentiate between different interactions

Reply:
Ok